buzzmobile/tests/sense/test_inputer.py

```diff
@@ -0,0 +1,55 @@
+import rostest
+import unittest
+import rospy
+import time
+from std_msgs.msg import String
+import roslaunch
+
+PKG = 'buzzmobile'
+NAME = 'test_inputer'
```

Comment:
Not really, note that name in this case is the node name for the test node, so it can be anything.

And PKG isn't provided anywhere, no, how could it be?